microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -1,150 +1,281 @@
-    *   Inspect if the `pv (REG_SZ)` regkey exists and is not `null` or empty.  Find  `pv (REG_SZ)` at the following location.  
+1.  During your app setup, test whether the WebView2 Runtime is already installed, using either of the following approaches:
+
+    *   Check whether the `pv (REG_SZ)` regkey exists and whether the regkey is `null` or empty.  If this regkey doesn't exist, or it is `null` or an empty string, the WebView2 Runtime isn't currently installed on the client.  Find `pv (REG_SZ)` at the following location:
         
-        On 64-bit Windows  
+        On 64-bit Windows:
         
         ```text
         HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
         ```  
         
-        On 32-bit Windows  
+        On 32-bit Windows:
         
         ```text
         HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
         ```  
         
-    *   Run [GetAvailableCoreWebView2BrowserVersionString][ReferenceWin32Webview2IdlGetavailablecorewebview2browserversionstring] and ensure the `versionInfo` is `NULL`.  
-1.  If the Runtime is not installed, run the standalone installer.  If you want to run a silent installation, either run the installer from an elevated process or copy and run the following command.  
+    *   Alternatively, call [GetAvailableCoreWebView2BrowserVersionString][ReferenceWin32Webview2IdlGetavailablecorewebview2browserversionstring] and check whether the `versionInfo` is `NULL`.  If `versionInfo` is `NULL`, the WebView2 Runtime isn't currently installed on the client.
+
+1.  If the WebView2 Runtime is not installed, run the Evergreen Standalone Installer.  If you want to run a silent installation, either run the installer from an elevated process, or copy and run the following command:
     
     ```shell
     MicrosoftEdgeWebView2RuntimeInstaller{X64/X86/ARM64}.exe /silent /install
     ```  
-    
-### Stay compatible in Evergreen mode  
 
-The Web is constantly evolving.  The Evergreen WebView2 Runtime is kept up to date to provide you with the latest features and security fixes.  To ensure your app stays compatible with the web, you should set up testing infrastructure.  
+### Test your app for forward-compatibility
+
+The Web is constantly evolving.  In the Evergreen distribution mode, the WebView2 Runtime is automatically kept up to date on the client to provide the latest features and security fixes.  If you use Evergreen distribution, to ensure that your WebView2 app stays compatible with the web, you should set up testing infrastructure.  
+
+Microsoft Edge preview channels \(Beta, Dev, and Canary\) provide a sneak peek into what is coming next in the WebView2 Runtime.  Test your WebView2 app regularly against a  Microsoft Edge preview channel, and update your app or [report issues][GithubMicrosoftedgeWebviewfeedback] if issues arise.  Typically, Beta and Dev are the recommended channels.
```

Comment:
@jasonstephen15 - For prerelease, this distribution.md article recommends Beta or Dev channels.  But Versioning article recommends Canary channel.  Which is recommended?
The published articles:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution - "Non-stable Microsoft Edge channels (Beta/Dev/Canary) provide a sneak peek into what is coming next into WebView2 Runtime. ... Test your WebView2 app against one of the non-stable channels ... Typically **Dev and Beta** are the recommended channels"

https://docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes - "The WebView team recommends that you use the **Canary** channel when you develop using the prerelease packages"